packages/mdc-checkbox/_mixins.scss

```diff
@@ -141,7 +141,7 @@
       @include mdc-checkbox__native-control_;
     }
 
-    &:disabled {
+    &[disabled] {
```

Comment:
If this is edge bug in styling checkbox can we add attribute selector only to mdc-checkbox?

`:disabled` CSS3 pseudo selector seems to be working for all other components in all major browsers.

WDYT?

Reply:
I didn't test the other components, I just figured they could have similar bugs. In order to really test it, you have to have a way to change its disabled state without interacting with the element itself. (e.g. put a button on the page that on click programmatically changes the disabled state of the component). If the component just starts off disabled, or you disable it by clicking on it, the bug won't show.

If we're confident other components don't have the issue I can switch them back, but it seems safer to just change them

Reply:
I'll test them out, to verify

Reply:
I just tested this on Edge browser programmatically toggling the button / checkbox and also rendering widgets with disabled state. I'm not able to reproduce the original issue?

![checkbox_disabled_edge](https://user-images.githubusercontent.com/47312/56153331-e80c8580-5f83-11e9-8038-60b6c3a5209f.gif)

https://marshy-tiger.glitch.me/

This MS Edge bug seems related:
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11295231/

Reply:
Good find, I've updated it to only change places where `:disabled` was being used together with the sibling combinator (which were all in checkbox).